Super Tafani Member/index.html

```diff
@@ -0,0 +1,220 @@
+                    <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M10 0C4.48 0 0 4.48 0 10C0 15.52 4.48 20 10 20C15.52 20 20 15.52 20 10C20 4.48 15.52 0 10 0ZM2 10C2 9.39 2.08 8.79 2.21 8.22L6.99 13V14C6.99 15.1 7.89 16 8.99 16V17.93C5.06 17.43 2 14.07 2 10ZM15.89 15.4C15.63 14.59 14.89 14 13.99 14H12.99V11C12.99 10.45 12.54 10 11.99 10H5.99V8H7.99C8.54 8 8.99 7.55 8.99 7V5H10.99C12.09 5 12.99 4.1 12.99 3V2.59C15.92 3.77 18 6.65 18 10C18 12.08 17.19 13.98 15.89 15.4Z" fill="white"/>
+                        </svg>
+                        
+                </div>
+                     
+               </li>
+               <li class="nav-item d-flex align-items-center mx-3">
+              
+                    
+                <div>
+                 <a class="nav-link active text-white full-height-border" aria-current="page" href="#" >Nada Elshaeer</a>
+                </div>
+                <div>
+                 <svg width="12" height="8" viewBox="0 0 12 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                     <path d="M10.9997 1.16994C10.8123 0.983692 10.5589 0.87915 10.2947 0.87915C10.0305 0.87915 9.77707 0.983692 9.5897 1.16994L5.9997 4.70994L2.4597 1.16994C2.27234 0.983692 2.01889 0.87915 1.7547 0.87915C1.49052 0.87915 1.23707 0.983692 1.0497 1.16994C0.955976 1.26291 0.881582 1.37351 0.830813 1.49537C0.780044 1.61723 0.753906 1.74793 0.753906 1.87994C0.753906 2.01195 0.780044 2.14266 0.830813 2.26452C0.881582 2.38638 0.955976 2.49698 1.0497 2.58994L5.2897 6.82994C5.38267 6.92367 5.49327 6.99806 5.61513 7.04883C5.73699 7.0996 5.86769 7.12574 5.9997 7.12574C6.13172 7.12574 6.26242 7.0996 6.38428 7.04883C6.50614 6.99806 6.61674 6.92367 6.7097 6.82994L10.9997 2.58994C11.0934 2.49698 11.1678 2.38638 11.2186 2.26452C11.2694 2.14266 11.2955 2.01195 11.2955 1.87994C11.2955 1.74793 11.2694 1.61723 11.2186 1.49537C11.1678 1.37351 11.0934 1.26291 10.9997 1.16994Z" fill="white"/>
+                     </svg>
+                </div>
+                     
+               </li>
+            </ul>
+            
+          </div>
+        </div>
+      </nav>
+   </header>
+   <main >
+    
+      <div class="container-fluid">
+        <div class="row ">
+            <aside class="col-2 side px-0 primary-background-bg  min-vh-100" >
+                <ul>
+                 <li class="text-white nav-item mt-60 px-30" ><a href="#Dashboard" class="nav-link">Dashboard</a></li>
+                 <li class="text-white nav-item mt-22 px-30"><a href="#My Customers" class="nav-link">My Customers</a></li>
+                 <li class="text-white nav-item mt-34 px-30"><a href="#Community" class="nav-link">Community</a></li>
+                 <li class="text-white nav-item mt-28 px-30"><a href="#Leaderboard" class="nav-link">Leaderboard</a></li>
+                 <li class="text-white nav-item mt-28 px-30"><a href="#Products" class="nav-link">Products</a></li>
```

Comment:
Please fix this typo it is Products & Services not Products

Reply:
done